test/python/WMCore_t/Services_t/DBS_t/DBSUtils_t.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+"""
+_DBSUtils_t_
+
+Unit test for the DBSUtils module
+"""
+
+import time
+import unittest
+
+from WMCore.Services.DBS.DBSUtils import urlParams
+from WMCore.Services.DBS.DBS3Reader import DBS3Reader
+
+
+class DBSUtilsTest(unittest.TestCase):
+
+    def testUrlParams(self):
+        """
+        urlParams should return dictionary of URL parameters
+        """
+        url = 'http://a.b.com?d=1&f=bla'
+        results = urlParams(url)
+        self.assertTrue(results.get('d'), 1)
+        self.assertTrue(results.get('f'), 'bla')
+        return
+
+
+    def testGetParallel(self):
```

Comment:
It's great to have a test like this, even if it's not run in jenkins. At least we can make sure our code is sound!

To avoid triggering this by every single jenkins run, can you please:
```
from nose.plugins.attrib import attr
```

and decorate this test with:
```
@attr("integration")
```
?

Reply:
ok